lib/foreman_remote_execution_core/fake_script_runner.rb

```diff
@@ -1,6 +1,7 @@
 module ForemanRemoteExecutionCore
   class FakeScriptRunner < ForemanTasksCore::Runner::Base
-
+    DEFAULT_REFRESH_INTERVAL = 60
```

Comment:
Shouldn't the `FakeScriptRunner` use the same default interval as a regular `ScriptRunner`, i.e. 1 second?

Reply:
+1 on 1 second